src/credential/issuance/__tests__/04-complete-user-authorization.test.ts

```diff
@@ -51,44 +26,30 @@ describe("authorizeUserWithQueryMode", () => {
       error_description: "123456",
     });
 
-    const authContext: AuthorizationContext = {
-      authorize: jest
-        .fn()
-        .mockResolvedValue(`test://example.com?${authErr.toString()}`),
-    };
+    const authRedirectUrl = `test://cb?${authErr.toString()}`;
 
-    await expect(() =>
-      completeUserAuthorizationWithQueryMode(
-        issuerRequestUri,
-        clientId,
-        issuerConf,
-        idpHint,
-        redirectUri,
-        authContext
-      )
-    ).rejects.toThrowError(AuthorizationIdpError);
+    const query = parseUrl(authRedirectUrl).query;
+
+    try {
+      await parseAuthroizationResponse(query);
+    } catch (error) {
+      expect(error).toBeInstanceOf(AuthorizationIdpError);
+    }
   });
 
   it("should throw an AuthorizationError when the authorization response is not recognized", async () => {
     const wrongAuthRes = new URLSearchParams({
       random: "abcdefg",
     });
 
-    const authContext: AuthorizationContext = {
-      authorize: jest
-        .fn()
-        .mockResolvedValue(`test://example.com?${wrongAuthRes.toString()}`),
-    };
+    const authRedirectUrl = `test://cb?${wrongAuthRes.toString()}`;
+
+    const query = parseUrl(authRedirectUrl).query;
 
-    await expect(() =>
-      completeUserAuthorizationWithQueryMode(
-        issuerRequestUri,
-        clientId,
-        issuerConf,
-        idpHint,
-        redirectUri,
-        authContext
-      )
-    ).rejects.toThrowError(AuthorizationError);
+    try {
+      await parseAuthroizationResponse(query);
```

Comment:
Same as above.

Reply:
It is an oversight, thank you for noticing. I will correct it